auth-web/src/components/auth/staff/continuation-in/StaffContinuationInTable.vue

```diff
@@ -0,0 +1,310 @@
+                        :id="header.value"
+                        input
+                        type="search"
+                        autocomplete="off"
+                        class="text-input-style"
+                        filled
+                        :placeholder="header.text"
+                        dense
+                        hide-details="auto"
+                      />
+                    </th>
+                  </tr>
+                </thead>
+              </template>
+
+              <!-- Item Actions -->
+              <template #[`item.action`]="{ item }">
+                <div class="actions text-right">
+                  <span class="open-action">
+                    <v-btn
+                      color="primary"
+                      :class="['open-action-btn', { active: isActiveAccounts }]"
+                      :data-test="getIndexedTag('view-account-button', item.id)"
+                      @click="view()"
+                    >
+                      {{ getButtonLabel(item.status) }}
+                    </v-btn>
+                  </span>
+                </div>
+              </template>
+            </v-data-table>
+          </transition>
+        </v-col>
+      </v-row>
+    </v-form>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive } from '@vue/composition-api'
+import BusinessServices from '@/services/business.services'
+
+export default defineComponent({
+  name: 'StaffAccountsTable',
+  setup () {
+    const state = reactive({
+      businesses: [],
+      headerAccounts: [
+        { text: 'Date Submitted', value: 'date' },
+        { text: 'NR Number', value: 'nrNumber' },
+        { text: 'Identifying Number', value: 'businessIdentifier' },
+        { text: 'Completing Party', value: 'completingParty' },
+        { text: 'Status', value: 'status' },
+        { text: 'Actions', value: 'action' }
+      ],
+      isTableLoading: false,
+      paginationOptions: [5, 10, 15, 20]
+    })
+
+    function getButtonLabel (status) {
+      const reviewStates = ['Awaiting Review', 'Resubmitted']
+      return reviewStates.includes(status) ? 'Review' : 'View'
+    }
+
+    function view () {
+      // To be updated: route to Continuation Authorization Review page
+      // root.$router.push()
+    }
+
+    function getIndexedTag (tag: string, index: number): string {
+      return `${tag}-${index}`
+    }
+
+    return {
+      ...state,
+      getButtonLabel,
+      view,
+      getIndexedTag
+    }
+  },
+  mounted () {
+    this.isTableLoading = true
+    BusinessServices.fetchContinuationBusinesses()
+      .then(result => {
+        this.businesses = result.data
+      })
+      .catch(error => {
+        console.error('Failed to fetch businesses:', error)
+      })
+      .finally(() => {
+        this.isTableLoading = false
+      })
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/scss/theme.scss';
+// Note this uses .fas-search
```

Comment:
Do a search for "FAS" and delete/rename accordingly in your components/modules.

Reply:
> Do a search for "FAS" and delete/rename accordingly in your components/modules.

May I know what "FAS" means? It seems "FAS" is just used in line 3:
<v-form class="fas-search continuation-application-search">

but this style is needed, for the search field

Reply:
FAS is Fee Accounting System -- a different product / repo / team.

Just rename this class then.

Reply:
Renamed